packages/api-explorer/src/lib/filter-empty-formdata.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2019-present Mia s.r.l.
+ * All rights reserved
+ */
+
+const isSchemaUsable = schema => schema && Object.keys(schema).length > 0
+
+export function isRequired(key, schema) {
+  if (!isSchemaUsable(schema)) {
+    return false
+  }
+
+  if (Array.isArray(schema.required) && schema.required.includes(key)) {
+    return true
+  }
+
+  return false
+}
+
+const getSubSchema = (key, schema) => {
+  if (!isSchemaUsable(schema)) {
+    return {}
+  }
+
+  if (schema.type === 'object' && schema.properties && schema.properties[key] && (schema.properties[key].type === 'object' || schema.properties[key].type === 'array')) {
+    return schema.properties[key]
+  }
+
+  return schema
+}
+
+const allUndefined = o => Object.entries(o).values(v => v !== undefined).length <= 0
```

Comment:
Why you use Object.entries(...).values(...)?

Reply:
`Object.values(o).map`